test/helpers/password.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 // The password used for integration tests:
-let password = 'CozyTest$1'
+let password = process.env.COZY_PASSPHRASE || 'CozyTest_1'
```

Comment:
The goal of this "executable" module was for the default test password/passphrase to be defined in a single place, but now it is defined thrice.